fbfmaproom/fbfmaproom.py

```diff
@@ -444,7 +444,11 @@ def select_forecast(country_key, forecast_key, issue_month0, target_month0,
             raise NotFoundError(f'No forecast for issue_month0 {issue_month0} in year {target_year}') from None
 
     if freq is not None:
-        da = da.sel(pct=freq)
+        if forecast_key.startswith("poe"):
```

Comment:
For the record, this (different behavior depending on the name we happen to choose for the forecast key) sucks. Merging this version in a hurry, hoping to clean up later.